dialogue-target/src/main/java/com/palantir/dialogue/BodySerDe.java

```diff
@@ -40,6 +48,11 @@
      * <p>
      * This method is named <pre>deserializeInputStream</pre> not <pre>deserializeBinary</pre>
      * to support future streaming binary bindings without conflicting method signatures.
+     *
+     * @deprecated Prefer {@link #inputStreamDeserializer()}
      */
-    InputStream deserializeInputStream(Response response);
+    @Deprecated
+    default InputStream deserializeInputStream(Response response) {
```

Comment:
Why not just get rid of this

Reply:
our circular dependency on generated code would break